llvm/lib/SYCLLowerIR/LowerWGLocalMemory.cpp

```diff
@@ -84,6 +86,42 @@ ModulePass *llvm::createSYCLLowerWGLocalMemoryLegacyPass() {
   return new SYCLLowerWGLocalMemoryLegacy();
 }
 
+// In sycl header __sycl_allocateLocalMemory builtin call is wrapped in
```

Comment:
Why can we not rewrite the SYCL headers to 'inline' these calls? Is there a specific reason? Thanks